### DISCUSSION TITLE
希望能出一个直播间进场提示消息屏蔽的功能

### DISCUSSION THREAD

**Discussion by Yur1et** [ywmoyue/biliuwp-lite#572] [Q&A] [answered]
### 问题描述

在人流量比较大的直播间，经常能看见大量进场提示，影响弹幕列表的阅读。希望能有一个选项，以便直接屏蔽所有进场消息提示。
![PixPin_2024-04-10_17-13-27](https://github.com/ywmoyue/biliuwp-lite/assets/90443238/85312e2b-cca7-49ca-b98b-4d15458f1f6d)

### 描述解决方案

希望右侧只显示弹幕信息

### 代码贡献

否

**Comment by GD-Slime** [comment, marked as answer]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/82302542/abf0a2f5-e20c-4612-99d6-e71c3142ed97)
你好 这个功能是有的哦
~(还能看到蒙神打职业吗😭)~

**Reply by Yur1et** [reply]
谢谢您，btw最近节奏真是没眼看
